Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'bcrypt-ruby'
 gem 'omniauth-twitter'
+gem 'tzinfo-data'
```

Comment:
It's in my command prompt when I run "rails s". It attempts to run it and then says Exiting...C:/RailsInstaller/Ruby2.1.0/lib/ruby/gems/2.1.0/gems/tzinfo-1.2.2/lib/tzinfo/data_source.rb:182:in 'rescue in create_default_data_source' : No source of timezone data could be found. (TZInfo::DataSourceNotFound)
Please refer to http://tzinfo.github.io/datasourcenotfound for help resolving this error.

From there I get about 60 lines of from C:/RailsInstaller/Ruby2.1.0/lib/ruby/gems/.......

I can take a screenshot if you need more.